app/contacts/[name]/page.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
+import React from "react";
+import { useParams } from "next/navigation";
+import { Contacts, contacts } from "../../Data/page";
+
+export default function Individual() {
```

Comment:
every function at minimum should be commented with what it does, parameters, and return type